packages/core/src/utils/collect-updates/collect-updates.ts

```diff
@@ -1,5 +1,5 @@
 import log from 'npmlog';
-import { ExecOpts, UpdateCollectorOptions } from '../../models';
+import {DescribeRefOptions, ExecOpts, UpdateCollectorOptions} from '../../models';
```

Comment:
are you using VSCode? I would assume that Prettier would add spaces around curly braces here, that is if you have the Prettier VSCode extension installed, if not please add spaces on both end of import